CMakeLists.txt

```diff
@@ -1076,6 +1087,30 @@ if (ENABLE_CGAL)
 endif()
 list(APPEND Sources src/openscad.cc ${CORE_SOURCES} ${OFFSCREEN_SOURCES})
 
+set_source_files_properties(
+  ${OFFSCREEN_SOURCES}
+  ${PLATFORM_SOURCES}
+  ${FLEX_openscad_lexer_OUTPUTS}
+  ${BISON_openscad_parser_OUTPUTS}
+  ${FLEX_comment_lexer_OUTPUTS}
+  ${BISON_comment_parser_OUTPUTS}
+  ${MANIFOLD_SOURCES}
+  src/glview/OffscreenContextFactory.cc
+  src/glview/RenderSettings.cc
+  src/glview/Camera.cc
+  src/glview/ColorMap.cc
+  src/glview/preview/CSGTreeNormalizer.cc
+  src/gui/input/SpaceNavInputDriver.cc
+  src/gui/QGLView.cc
+  src/gui/QGLView2.cc
+  # things that requires _USE_MATH_DEFINES
```

Comment:
I do wonder if it's worthwhile porting off math defines, e.g. to boost constants (https://www.boost.org/doc/libs/1_37_0/libs/math/doc/sf_and_dist/html/math_toolkit/toolkit/internals1/constants.html), or collect in our own header while we wait for C++20.